plugin/core/sessions.py

```diff
@@ -682,6 +682,16 @@ def name(cls) -> str:
         """
         raise NotImplementedError()
 
+    @classmethod
+    def status_name(cls) -> str:
```

Comment:
this is similar to lsp_utils `get_displayed_name`?
https://github.com/sublimelsp/lsp_utils/blob/master/st3/lsp_utils/_client_handler/interface.py#L29

Reply:
Based on the `name` method description,
it sounds like the `name` method already does what `status_name` wants to do:
```
    @classmethod
    @abstractmethod
    def name(cls) -> str:
        """
        A human-friendly name. If your plugin is called "LSP-foobar", then this should return "foobar". If you also
        have your settings file called "LSP-foobar.sublime-settings", then you don't even need to re-implement the
        configuration method (see below).
        """
        raise NotImplementedError()
```

Reply:
Your comment makes me think that you haven't read the description I gave in the initial comment. :)

As for `get_displayed_name()`, it overrides `name()` under the hood.

Reply:
But to clarify, this only affects the name shown in the status field. `name()` affects a lot more (configuration keys, name in LSP logs, error messages and possibly more).